api/src/main/java/org/openmrs/Order.java

```diff
@@ -363,6 +363,38 @@ public void setCommentToFulfiller(String commentToFulfiller) {
 	}
 	
 	/**
+	 * Convenience method to determine if the order is activated as of the current date
+	 * 
+	 * @return boolean indicating whether the order was activated before or on the current date
+	 * @since 2.0
+	 * @see #isActivated(java.util.Date)
```

Comment:
@wluyima if I should do so, can you please be specific in how I should do this? Extracting new Date() into an interface method so it can be mocked during testing? Yes, no? Is this worth the effort?